mobile/lib/authInterceptor.ts

```
@@ -0,0 +1,85 @@
+};
+
+export const attachAuthInterceptors = (client: AxiosInstance, handlers: AuthHandlers) => {
+  let isRefreshing = false;
+  let refreshSubscribers: { resolve: (token: string) => void; reject: (err: unknown) => void }[] = [];
+
+  const subscribeTokenRefresh = (resolve: (token: string) => void, reject: (err: unknown) => void) => {
+    refreshSubscribers.push({ resolve, reject });
+  };
+
+  const notifySubscribers = (token: string) => {
+    refreshSubscribers.forEach(({ resolve }) => resolve(token));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)